cmd/restic/cmd_sync.go

```diff
@@ -139,6 +142,8 @@ func runSync(opts SyncOptions, gopts GlobalOptions, term *termstatus.Terminal, a
 	}
 
 	p := ui.NewBackup(term, gopts.verbosity)
+	// Hack to disable status line as it causes problems by listing every file
+	p.MinUpdatePause = time.Hour
```

Comment:
Can you try adjusting term := termstatus.New(globalOptions.stdout, globalOptions.stderr, globalOptions.Quiet) on line 35 to just take a true for the last param and see how that works?

Reply:
Ooh, great catch -- that should work and may fix another issue I was looking into